tuner/tuner/libtuner.py

```diff
@@ -1059,24 +1052,23 @@ def benchmark(
     if not baseline_handler.is_valid():
         logging.warning("Baseline run failed.")
 
-    speedup_result = baseline_handler.calculate_speedup(candidate_results)
-    all_candidates_with_speedup = baseline_handler.sort_candidates_with_speedup(
-        speedup_result
+    all_candidates_with_speedup = baseline_handler.get_candidates_ordered_by_speedup(
+        candidate_results
     )
     top_candidates_with_speedup = all_candidates_with_speedup[:num_candidates]
 
     if baseline_handler.is_valid():
-        candidate_time_map = {
+        candidate_id_to_time_ms = {
             result.candidate_id: result.time for result in candidate_results
         }
         for candidate_id, speedup in top_candidates_with_speedup:
-            actual_time = candidate_time_map[candidate_id]
+            time_ms = candidate_id_to_time_ms[candidate_id]
```

Comment:
If `get_candidates_ordered_by_speedup` returned `list[tuple[BenchmarkResult, float]]:`, we could get rid of this logic.